testcontainers/src/core/image.rs

```diff
@@ -149,6 +149,7 @@ impl ImageArgs for () {
 pub struct RunnableImage<I: Image> {
     image: I,
     image_args: I::Args,
+    image_name: Option<String>,
```

Comment:
Or, as an alternative, we could add optional `regitstry`, and prepend it for descriptor.

But since we don't have such config on `Image` level - it seems unreasonable without changing `Image` API

Reply:
Added PR with alternative solution https://github.com/testcontainers/testcontainers-rs/pull/550,

WDYT @thomaseizinger? Which one do you prefer (if any)? My vision is described within #335 issue

Reply:
I've responded in the issue.